.github/workflows/linters.yml

```diff
@@ -33,7 +33,7 @@ jobs:
           npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
           [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/ror/.stylelintrc.json
```

Comment:
### Optional
- [ ] It would be better if we have a root route, it could be a sign in page, or a landing page if we want. But it is always great to have a root route for a better UI/UX